invenio_vocabularies/cli.py

```diff
@@ -29,6 +29,8 @@ def _process_vocab(config, num_samples=None):
         readers_config=config["readers"],
         transformers_config=config.get("transformers"),
         writers_config=config["writers"],
+        batch_size=config.get("batch_size", 1000),
```

Comment:
```suggestion
        batch_size=config.get("batch_size", 100),
```

minor: just because below I see the batch size being 100 by default. Otherwise 1000 should be applied everywhere? Or it could be I've misunderstood, and the 100 is just for chunking the 1000-sized batch, in which case ignore this comment completely :)

Reply:
It's the second case you described, `batch_size` is to determine the size of batch that is processed from reading to writting, while the 100 is just to chunk only the read process in smaller btaches (with this it seems to use a bit less memory)